app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,35 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts.order(created_at: :desc)
+    @current_user = current_user
   end
 
   def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
   end
+
+  def new
+    @current_user = current_user
+  end
```

Comment:
- [ ] the `new` method in the `PostController` should look something like below:
```
    def new
      @post = Post.new
    end

```